biome.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://biomejs.dev/schemas/2.1.2/schema.json",
```

Comment:
Iff we'd switch linter, it would have to be in a completely separate PR without any added features, and we'd have to bike-shed that for a while on Discord first.

Reply:
> Any reason why?

I added a few comments here or there, but I won't be able to give a thorough review if your actual changes are interleaved with a ton of changes from replacing the linter, that's an unreasonable ask IMO.

Reply:
Maybe it's better to close the PR then?

Reply:
I don't want to discourage you by blatantly closing the PR - I'll let you decide its faith.

I would love to discuss and merge many of the improvements, if we're able to get them reviewed one-by-one 👍